restcomm/restcomm.docs/sources-asciidoc/src/main/asciidoc/configuration/docker/Restcomm - Docker Quick Start Guide.adoc

```diff
@@ -1,69 +1,53 @@
+* *SELinux:* This has been known to cause "permission denied" issues on some Linux system, you might want to disable or set SELinux to permissive mode.* (_hint_: getenforce, will show the type of permission you currently have )
 
-To pull the "**latest**" image from the RestComm docker hub repository please use:
-
-[source,lang:default,decode:true]
-----
-docker pull restcomm/restcomm:latest
-----
+[[run-restcomm]]
+Run RestComm
+^^^^^^^^^^^^
 
-[[install-and-run-restcomm-docker]]
-Install and Run Restcomm Docker
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+RestComm-Connect includes a single `docker-compose.yml` file, which is meant to document how the RestComm Docker image should be deployed.
 
-* Get your "ETHERNET_IP" by running ifconfig. Use 127.0.0.1 as "ETHERNET_IP" if you are running container on Docker for Mac.
-* Fill out the RCBCONF_STATIC_ADDRESS variable as shown below.
-* Start Restcomm as shown below:
-* This will start Restcomm in secure mode (https)
+It comes with sane defaults for a standalone development environment, so it should pretty much work out of the box, if you set the correct value for `RCBCONF_STATIC_ADDRESS` and `RCBCONF_VOICERSS_KEY`.
 
-[source,lang:default,decode:true]
+To try it out, change to the folder where `docker-compose.yml` is located and run:
+[source:bash]
 ----
-docker run  -i -d --name=restcomm-myInstance -v /var/log/restcomm/:/var/log/restcomm/ -e RCBCONF_STATIC_ADDRESS="YOUR_ETHERNET_IP" -e ENVCONFURL="https://raw.githubusercontent.com/RestComm/Restcomm-Docker/master/env_files/restcomm_env_locally.sh" -p 8080:8080 -p 8443:8443 -p 9990:9990 -p 5080:5080 -p 5081:5081 -p 5082:5082 -p 5083:5083 -p 5080:5080/udp -p 65000-65050:65000-65050/udp restcomm/restcomm:latest
+$ docker-compose up
 ----
 
-* OS X seems to have known for mount denied issue on `/var/log/restcomm` directory, in such case: mount logs with any directory under `/Users`. Here is an example:
 
-[source,lang:default,decode:true]
-----
-docker run  -i -d --name=restcomm-myInstance -v /Users/username/var/log/restcomm:/var/log/restcomm/ -e RCBCONF_STATIC_ADDRESS="YOUR_ETHERNET_IP" -e ENVCONFURL="https://raw.githubusercontent.com/RestComm/Restcomm-Docker/master/env_files/restcomm_env_locally.sh" -p 8080:8080 -p 8443:8443 -p 9990:9990 -p 5080:5080 -p 5081:5081 -p 5082:5082 -p 5083:5083 -p 5080:5080/udp -p 65000-65050:65000-65050/udp restcomm/restcomm:latest
-----
+[[supported-tags]]
+Supported Tags
+^^^^^^^^^^^^^^
 
+A note on the docker image tags:
 
-[[text-to-speech-with-voicerss]]
-Text-to-Speech with VoiceRSS
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+* Tag "latest". Points to the latest binary from the Continuous Delivery server. Uses the https://github.com/RestComm/Restcomm-Connect[master] development branch : restcomm/restcomm:latest. *We don't advice to use "latest" tag for production as it is constantly changing, and as well new features and fixes may not be documented.
+* Tag "7.7.0" pointing to current specific GA release (v770ga).
```

Comment:
The latest release is 8.2.1.GA in the community https://github.com/RestComm/Restcomm-Connect/releases/tag/8.2.1-67. Can we update the docker tags to match that ?

Reply:
@deruelle changed the wording here with commit https://github.com/RestComm/Restcomm-Connect/pull/2687/commits/cd4a06f14ac9599745a517a2659b78f006fd3481 , so this should be resolved.